src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  padding: 10px 20px;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
You can even try to use here  'grid-template-columns' to `repeat(auto-fit, minmax(200px, 1fr));` to make the layout responsive and just leave only this breakpoint to make sure then you get max 4 columns.

`@media all and (min-width: 1024px) {
    grid-template-columns: repeat(4, 200px);
  }`

Then you can remove the rest breakpoints for the smaller screens.

`min-width: 488px` and `min-width: 768px`